src/main/java/de/uni_mannheim/informatik/dws/wdi/WDI_Project_T2/util/Utils.java

```diff
@@ -48,8 +76,10 @@ private static String replaceUmlaute(String orig) {
 
     public static String preprocessModel(String modelDescription) {
         String cleaned = Utils.removeUnderscores(modelDescription);
+        cleaned = cleaned.toLowerCase();
         cleaned = Utils.replaceUmlaute(cleaned);
-        cleaned = Utils.removeNonDescriptionWords(cleaned);
+        //cleaned = Utils.removeNonDescriptionWords(cleaned);
```

Comment:
Remove the method and call completely

Reply:
This was Nadja's work; I didn't want to remove it, as it shows our progress of ideas approaching our solution ;-) I hence just commented out the part which calls this method.

Reply:
s.o.